Python/PlenoInterview/Problem1/order.py

```diff
@@ -0,0 +1,40 @@
+class order:
```

Comment:
The Class name should be `Order` as it's a Class, so the first letter is capitalized.